components/uniset/src/props.rs

```diff
@@ -3,24 +3,19 @@
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 #![allow(clippy::unreadable_literal, dead_code)]
 
+use crate::enum_props::*;
 use crate::provider::*;
 use crate::{UnicodeSet, UnicodeSetError};
 use icu_provider::prelude::*;
 use std::borrow::Cow;
 use std::convert::TryInto;
 
 // helper fn
-fn get_prop<'d, D: DataProvider<'d, UnicodeProperty<'d>> + ?Sized>(
-    ppucd_provider: &D,
-    resc_key: ResourceKey,
-) -> Result<UnicodeSet, UnicodeSetError> {
+fn get_prop<'d, D: DataProvider<'d, UnicodeProperty<'d>> + ?Sized>(ppucd_provider: &D, resc_key: ResourceKey) -> Result<UnicodeSet, UnicodeSetError> {
```

Comment:
`option`: for maintenance:

```rust
type Result = Result<UnicodeSet, UnicodeSetError>;

fn get_prop<'d, D: DataProvider<'d, UnicodeProperty<'d>> + ?Sized>(ppucd_provider: &D, resc_key: ResourceKey) -> Result {}
```

or even:

```rust
macro_rules! helper_fn {
    ($name:ident, $body:expr) => {
fn $name<'d, D: DataProvider<'d, UnicodeProperty<'d>> + ?Sized>(ppucd_provider: &D, resc_key: ResourceKey) -> Result<UnicodeSet, UnicodeSetError> {
    get_prop(provider, $key)
}
    };
}
```

and then:

```rust
helper_fn!(get_ascii_hex_digit_property, key::ASCII_HEX_DIGIT_V1);
helper_fn!(get_alnum_property, key::ALNUM_V1);
```

Reply:
Done.

Reply:
I see you didn't decide to go for the macro. Fine by me, but more hassle for maintenance later.